### PR TITLE
Publish updates from master

### PR DESCRIPTION
**Pull request by usha-mandya**
Publish the latest changes from master

### CODE REVIEW

File: desktop/windows/troubleshoot.md [hunk 1]
```diff
@@ -54,9 +54,10 @@ from the menu.
 4. When the diagnostics collection process is complete, click **Upload to get a Diagnostic ID**.
 5. When the diagnostics have been uploaded, Docker Desktop prints a Diagnostic ID. Copy this ID.
 6. If you have subscribed to a Pro or a Team plan, click **Contact Support**. This opens the [Docker Desktop support](https://hub.docker.com/support/desktop/){:target="_blank" rel="noopener" class="_"} form. Fill in the information required and add the ID you copied earlier to the Diagnostics ID field. Click **Submit** to request Docker Desktop support.
+
    > **Note**
-    >
-    > You must be signed in to Docker Desktop using your Pro or Team plan credentials to access the support form. For information on what's covered as part of Docker Desktop support, see [Support](#support).
+   >
+   > You must be signed in to Docker Desktop using your Pro or Team plan credentials to access the support form. For information on what's covered as part of           Docker Desktop support, see [Support](#support).
```
Comment (StefanScherer) [inline desktop/windows/troubleshoot.md:60]: Are the many spaces intended here? Sorry I missed to review the original PR.

Comment (usha-mandya) [reply desktop/windows/troubleshoot.md:60]: Thanks @StefanScherer. This is fixed in https://github.com/docker/docker.github.io/pull/13368

**Review by thaJeztah — APPROVED** [review]
LGTM

**Review by thaJeztah — APPROVED** [review]
still LGTM

**Pull request merged**